packages/codegen-ui-react/lib/forms/form-renderer-helper/relationship.ts

```diff
@@ -1024,14 +1025,25 @@ export const buildManyToManyRelationshipStatements = (
                             ),
                             undefined,
                             [
-                              getCreateJoinTableExpression(
-                                relatedJoinTableName,
-                                savedModelName,
-                                joinTableThisModelName,
-                                joinTableRelatedModelName,
-                                importCollection,
-                                dataApi,
-                              ),
+                              isGraphql && isCompositeKey
+                                ? getGraphQLJoinTableCreateExpression(
```

Comment:
GraphQL version should be usable even when not composite key, reusing the composite key logic in there would allow this to be resilient to custom keys as well

Reply:
Oh ok, going to look into supporting custom keys